lib/extensions/canvas.dart

```diff
@@ -0,0 +1,13 @@
+import 'vector2.dart';
+
+import 'dart:ui' show Canvas;
```

Comment:
Maybe we should also export it the `Canvas` class, the same we did with the vector extensions?

That way a developer can just use our `Canvas` without having to import it from `dart:ui` itself as well. And then we can internally update all our own `Canvas` imports to our own extension variant.